### PR TITLE
[Docs] Translate understand_mmcv about registry.md and fix config.md

### PR DESCRIPTION
**Pull request by AluminiumOxide**
add translation of registry.md and fix config.md

### CODE REVIEW

**Comment by zhouzaida** [comment]
Thanks for your translation. Please fix the lint.

File: docs_zh_CN/understand_mmcv/config.md [hunk 1]
```diff
@@ -1,3 +1,177 @@
 ## 配置
 
-欢迎有兴趣的朋友一起翻译 MMCV 文档。如有兴趣，请在 [MMCV issue](https://github.com/open-mmlab/mmcv/issues) 提 issue 确定翻译的文档。
+`Config` 类用于配置操作和配置文件，它支持从多种文件格式中加载配置，包括 **python**, **json** 和 **yaml**。
```
Comment (zhouzaida) [inline docs_zh_CN/understand_mmcv/config.md:3]: 这里可以修改一下
```suggestion
`Config`  类用于操作配置文件，它支持从多种文件格式中加载配置，包括 **python**, **json** 和 **yaml**。
```

File: docs_zh_CN/understand_mmcv/config.md [hunk 2]
```diff
@@ -1,3 +1,177 @@
...
+它提供了类似字典对象的接口来获取和设置值。
+
+这里根据控制文件`test.py`举例。
```
Comment (zhouzaida) [inline docs_zh_CN/understand_mmcv/config.md:6]:
```suggestion
举配置文件 `test.py` 为例。
```

File: docs_zh_CN/understand_mmcv/config.md [hunk 3]
```diff
@@ -1,3 +1,177 @@
...
+```python
+>>> cfg = Config.fromfile('test.py')
+>>> print(cfg)
+>>> dict(a=1,
+...      b=dict(b1=[0, 1, 2], b2=None),
+...      c=(1, 2),
+...      d='string')
+```
+
+对于所有格式的配置文件，都支持一些预定义变量。它会将在`{{ var }}`里的变量转换为实际值。
```
Comment (zhouzaida) [inline docs_zh_CN/understand_mmcv/config.md:26]:
```suggestion
对于所有格式的配置文件，都支持预定义变量。它会将 `{{ var }}` 替换为实际值。
```

File: docs_zh_CN/understand_mmcv/config.md [hunk 4]
```diff
@@ -1,3 +1,177 @@
...
+
+目前支持以下四个预定义变量：
+
+
+`{{ fileDirname }}` - 当前打开文件的文件目录名，例如 /home/your-username/your-project/folder
```
Comment (zhouzaida) [inline docs_zh_CN/understand_mmcv/config.md:31]:
```suggestion
`{{ fileDirname }}` - 当前打开文件的目录名，例如 /home/your-username/your-project/folder
```

File: docs_zh_CN/understand_mmcv/config.md [hunk 5]
```diff
@@ -1,3 +1,177 @@
...
+
+`{{ fileBasename }}` - 当前打开文件的文件名，例如 file.ext
+
+`{{ fileBasenameNoExtension }}` - 当前打开文件不包含扩展名的文件名，例如 file
+
+`{{ fileExtname }}` - 当前打开文件的扩展名，例如 .ext
+
+这些变量名引用自 [VS Code](https://code.visualstudio.com/docs/editor/variables-reference)。
+
+这里根据一个带预定义变量的控制文件举例。
```
Comment (zhouzaida) [inline docs_zh_CN/understand_mmcv/config.md:41]:
```suggestion
这里举一个带有预定义变量的配置文件为例。
```

File: docs_zh_CN/understand_mmcv/config.md [hunk 6]
```diff
@@ -1,3 +1,177 @@
...
+
+```python
+>>> cfg = Config.fromfile('./config_a.py')
+>>> print(cfg)
+>>> dict(a=1,
+...      b='./work_dir/config_a',
+...      c='.py')
+```
+
+对于所有格式的配置文件, 都支持继承。为了重用其他控制文件，
```
Comment (zhouzaida) [inline docs_zh_CN/understand_mmcv/config.md:58]:
```suggestion
对于所有格式的配置文件, 都支持继承。为了重用其他配置文件的字段，
```

File: docs_zh_CN/understand_mmcv/config.md (hunk 5 above)
Comment (zhouzaida) [reply docs_zh_CN/understand_mmcv/config.md:41]: config 在 mmcv 的语境下翻译为 配置 更为准确

File: docs_zh_CN/understand_mmcv/config.md [hunk 7]
```diff
@@ -1,3 +1,177 @@
...
+...      b='./work_dir/config_a',
+...      c='.py')
+```
+
+对于所有格式的配置文件, 都支持继承。为了重用其他配置文件的字段，
+需要指定 `_base_='./config_a.py'` 或者一个包含控制文件的列表 `_base_=['./config_a.py', './config_b.py']`。
+
+这里有 4 个配置继承关系的例子。
+
+`config_a.py` 作为基类控制文件
```
Comment (zhouzaida) [inline docs_zh_CN/understand_mmcv/config.md:63]:
```suggestion
`config_a.py` 作为基类配置文件
```

File: docs_zh_CN/understand_mmcv/config.md [hunk 8]
```diff
@@ -1,3 +1,177 @@
 ## 配置
 
-欢迎有兴趣的朋友一起翻译 MMCV 文档。如有兴趣，请在 [MMCV issue](https://github.com/open-mmlab/mmcv/issues) 提 issue 确定翻译的文档。
+`Config` 类用于操作配置文件，它支持从多种文件格式中加载配置，包括 **python**, **json** 和 **yaml**。
+它提供了类似字典对象的接口来获取和设置值。
+
+举配置文件 `test.py` 为例。
```
Comment (MeowZheng) [inline docs_zh_CN/understand_mmcv/config.md:6]: 以配置文件 `test.py` 为例。

File: docs_zh_CN/understand_mmcv/config.md [hunk 9]
```diff
@@ -1,3 +1,177 @@
...
+
+`{{ fileBasename }}` - 当前打开文件的文件名，例如 file.ext
+
+`{{ fileBasenameNoExtension }}` - 当前打开文件不包含扩展名的文件名，例如 file
+
+`{{ fileExtname }}` - 当前打开文件的扩展名，例如 .ext
+
+这些变量名引用自 [VS Code](https://code.visualstudio.com/docs/editor/variables-reference)。
+
+这里举一个带有预定义变量的配置文件为例。
```
Comment (MeowZheng) [inline docs_zh_CN/understand_mmcv/config.md:41]: 这里是一个带有预定义变量的配置文件的例子。

**Review by MeowZheng — APPROVED** [review]
LGTM

**Review by zhouzaida — APPROVED** [review]
LGTM

**Comment by OpenMMLab-Coodinator** [comment]
Hi @AluminiumOxide ！First of all, we want to express our gratitude for your significant PR in this project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**